config/config-template-mac.yaml

```diff
@@ -0,0 +1,61 @@
+# This is a regular template file, which shows only the most important configuration entries for running on MacOS and using token as authentication against the sevice you want to test.
+# Additional configuration options are available, as shown in "config-template-long.yaml".
+# All the values are optional (except `config.configVersion`): if a key is missing, it will mean either "disabled" or a sensible default will be selected
+
+config:
+  # WARNING: `configVersion` indicates the schema version of the config file.
+  # This value tells RapiDAST what schema should be used to read this configuration.
+  # Therefore you should only change it if you update the configuration to a newer schema
+  # It is intended to keep backward compatibility (newer RapiDAST running an older config)
+  configVersion: 4
+
+  # the following is a workaround until Rapidast v2.1.1 is out
+  defectDojo:
+    url: "http://localhost"
```

Comment:
The fix for this is already in the `development` branch. 
I think it might be better to remove the workaround, since it should work already.

Reply:
removed